persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -1,5 +1,9 @@
 {-# LANGUAGE DeriveDataTypeable #-}
+{-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GADTs #-}
```

Comment:
I copied over a method from `MySQL.hs` to work with `postgresql`, and all of these imports were needed.  Should I alphabetize them manually?

Reply:
You can run `stylish-haskell` and it should work to make it right.